lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/VortexWorkerManager.java

```diff
@@ -35,7 +35,7 @@
 class VortexWorkerManager {
```

Comment:
Thanks for the catch. This class itself is actually not threadsafe. It is just safely accessed thanks to the global lock in `RunningWorkers`. I changed the annotation to `@NotThreadSafe`.